pyproject.toml

```diff
@@ -12,16 +12,16 @@ readme = "README.rst"
 
 [tool.poetry.dependencies]
 carehare = "~=1.0"
-channels = "~=3.0"
+channels = ">=3.0,<5"
 msgpack = "~=1.0"
-python = "~=3.8"
+python = ">=3.8,<3.11"
 
 [tool.poetry.dev-dependencies]
-pytest = "~=6.1"
-pytest-asyncio = "~=0.14"
-pytest-cov = "~= 2.10"
-pytest-timeout = "~= 1.4"
-asgiref = "~=3.1"
+pytest = ">=7.1.3,<8"
```

Comment:
Is >=x,<y standard in the Python universe? I'm not very Pythonic these days. I thought ~= notation was the way to go.